backend.native/tests/runtime/basic/random.kt

```diff
@@ -13,51 +13,51 @@ import kotlin.test.*
 /**
  * Tests that setting the same seed make random generate the same sequence
  */
-private inline fun <reified T> testReproducibility(seed: Long, generator: () -> T) {
+private inline fun <reified T> testReproducibility(seed: Long, generator: Random.() -> T) {
```

Comment:
I've edited this test so that it compiles, but at this point it duplicates SeededRandomSmokeTest from common stdlib.

Reply:
So would you like this test to be removed then?